docs/guides/Redux-Integration.md

```diff
@@ -2,16 +2,16 @@
 
 To handle your app's navigation state in redux, you can pass your own `navigation` prop to a navigator. Your navigation prop must provide the current state, as well as access to a dispatcher to handle navigation options.
 
-With redux, your app's state is defined by a reducer. Each navigation router effectively has a reducer, called `getStateForAction`. The following is a minimal example of how you might use navigators within a redux application:
+With redux, your app's state is defined by a reducer. Each navigation router effectively has a reducer, called `getStateForAction`. You can dispatch `NavigationActions.init` to initialize your router state.
+
+The following is a minimal example of how you might use navigators within a redux application:
 
 ```
-import { addNavigationHelpers } from 'react-navigation';
+import { addNavigationHelpers, NavigationActions } from 'react-navigation';
 
 const AppNavigator = StackNavigator(AppRouteConfigs);
 
-const initialState = AppNavigator.router.getStateForAction(AppNavigator.router.getActionForPathAndParams('Login'));
```

Comment:
My main concern with this PR is that it's just a bit more complex to set up for first-time users. It's not a very typical redux pattern. I have it working in a medium-large app with drawer+tabs+stacks all nested and it works great for me with just the changes in #937 in my reducer.